File: services/dbservice/src/main/java/com/torin/dbService/dto/MessagesEntitiesDto.java
package com.torin.dbService.dto;

import java.time.Instant;

public record MessagesEntitiesDto(
        Long idGroup,
        Long isComments,
        Long idMessage,
        String type,
        Integer entityOffset,
        Integer length,
        String value,
        Instant date) {
}

File: services/telegramservice/DTOs/DTO/MessagesEntitiesDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class MessagesEntitiesDTO
    {
        public long? IdGroup { get; set; }
        public long? IdMessage { get; set; }
        public int? IsComments { get; set; }
        public string? Type { get; set; }
        public int? EntityOffset { get; set; }
        public int? Length { get; set; }
        public string? Value { get; set; }
        public DateTime? Date { get; set; }
    }
}